src/java/com/zimbra/qa/selenium/projects/admin/pages/WizardAddNewVolume.java

```diff
@@ -26,13 +26,13 @@
 public class WizardAddNewVolume extends AbsWizard {
 	
 	public static class Locators {
-		public static final String VOLUME_TYPE_LOCAL = "css=[style*='z-index: 7'] input[id='zdlgv__DLG_NEW_PWRSTR_VOLUME_BObject_49']";
+		public static final String VOLUME_TYPE_LOCAL = "css=input[id='zdlgv__DLG_NEW_PWRSTR_VOLUME_BObject_63']";
```

Comment:
Update this located it contains number in the ID at the end

Reply:
I have verified locator. It is constant and not changing changing on every login.